### ISSUE TITLE
Focus indicator on more options button in a selected tree item has insufficient contrast against its background

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#871]
The following AI-windows mode(s) have this issue:
- High Contrast White mode
- High Contrast Black mode

**Screenshots or .GIF**
#### Screenshot of the issue in High Contrast White mode
![18 1](https://user-images.githubusercontent.com/22068434/88356548-1cd9c180-cd1d-11ea-80e2-5f2323c3b95c.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by DaveTryon** [comment]
@iamrafan, this has been fixed in the release available at https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.1388.01. Please validate/close at your convenience. Thanks!

**Comment by iamrafan** [comment]
Validated the fixes in Canary Release: v1.1.1388.01. All 6 modes are now compliant
